docs/book/src/quick-start.md

```diff
@@ -24,64 +32,67 @@ sudo mv /tmp/kubebuilder_2.0.1_${os}_${arch} /usr/local/kubebuilder
 export PATH=$PATH:/usr/local/kubebuilder/bin
 ```
 
-You can also install a KubeBuilder master snapshot from
-`https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
-
-Install [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
+**Note:** Also, you can install a master snapshot from `https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
 
 ## Create a Project
 
-Initialize a new project and Go module for your controllers:
+Create a directory then run the following command inside of it to initialize a new project:
 
 ```bash
 kubebuilder init --domain my.domain
 ```
 
-If you're not in `GOPATH`, you'll need to run `go mod init <modulename>`
+**Note:** If you're not in `GOPATH`, you'll need to run `go mod init <modulename>`
 in order to tell kubebuilder and Go the base import path of your module.
 
 ## Create an API
 
-Create a new API group-version called `webapp/v1`, and a kind `Guestbook`
-in that API group-version:
+* Run the following command to create a new API (group/version) as `webapp/v1` and the new Kind(CRD) `Guestbook` on it:
 
 ```bash
 kubebuilder create api --group webapp --version v1 --kind Guestbook
 ```
 
-This will create the files `api/v1/guestbook_types.go` and
-`controller/guestbook_controller.go` for you to edit.
+**Note:** If you press `y` for Create Resource [y/n] and for Create Controller [y/n] then this will create the files `api/v1/guestbook_types.go` where the API is defined 
```

Comment:
same before about a proper note block.  Look for `class="note` in the documentation to find an example.

Reply:
👍